src/content/docs/dropins/payment-services/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Payment Services overview
+description: Learn about the features of the Payment Services drop-in component.
+---
+
+import Link from '@components/Link.astro';
+import List from '@components/List.astro';
+
+The Payment Services drop-in renders the credit card form, allowing shoppers to efficiently enter their payment details. The drop-in currently supports credit/debit cards only.
+
+## Available Payment Services
+
+The Payment Services drop-in component consists of currently one container: `CreditCard`:
+
+- **Credit card container**: The `CreditCard` container is designed to allow placing an order with the available credit card details during the checkout process.
+
+## Additional Resources
+
+- [Payment Service API Documentation](https://developer.adobe.com/commerce/webapi/graphql/payment-services-extension/)
+- [Integration Examples](https://developer.adobe.com/commerce/webapi/graphql/payment-services-extension/workflows/)
```

Comment:
idem - not a name
```suggestion
- [Integration examples](https://developer.adobe.com/commerce/webapi/graphql/payment-services-extension/workflows/)
```

apart from the case, what does "Integration examples" mean, here? could we call it "Payment Services workflows", or similar?